<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-EER3MNQ" failures="1" tests="13" name="testCases.S001LogInCases" time="38.284" errors="0" timestamp="2021-03-04T00:56:25 ART" skipped="3">
  <testcase classname="testCases.S001LogInCases" name="welcomeisDisplayed" time="0.108">
    <failure message="The following asserts failed:
	expected [WELCOME TO MYXfitness!dsedsf] but found [WELCOME TO MYXfitness!]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [WELCOME TO MYXfitness!dsedsf] but found [WELCOME TO MYXfitness!]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at testCases.S001LogInCases.welcomeisDisplayed(S001LogInCases.java:50)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
    </failure>
  </testcase> <!-- welcomeisDisplayed -->
  <system-out/>
  <testcase classname="testCases.S001LogInCases" name="welcomeisDisplayed" time="0.110">
    <skipped/>
  </testcase> <!-- welcomeisDisplayed -->
  <system-out/>
  <testcase classname="testCases.S001LogInCases" name="welcomeisDisplayed" time="0.119">
    <skipped/>
  </testcase> <!-- welcomeisDisplayed -->
  <system-out/>
  <testcase classname="testCases.S001LogInCases" name="welcomeisDisplayed" time="0.244">
    <skipped/>
  </testcase> <!-- welcomeisDisplayed -->
  <system-out/>
  <testcase classname="testCases.S001LogInCases" name="theRoadisDisplayed" time="0.171"/>
  <system-out/>
  <testcase classname="testCases.S001LogInCases" name="invalidEmail" time="5.989"/>
  <system-out/>
  <testcase classname="testCases.S001LogInCases" name="invalidPassword" time="4.339"/>
  <system-out/>
  <testcase classname="testCases.S001LogInCases" name="noEmailAndPassword" time="3.157"/>
  <system-out/>
  <testcase classname="testCases.S001LogInCases" name="passwordRecoveryEmpty" time="3.654"/>
  <system-out/>
  <testcase classname="testCases.S001LogInCases" name="passwordRecoveryWrongEmail" time="5.928"/>
  <system-out/>
  <testcase classname="testCases.S001LogInCases" name="passwordRecoveryzValid" time="5.519"/>
  <system-out/>
  <testcase classname="testCases.S001LogInCases" name="iDontHaveAccount" time="2.636"/>
  <system-out/>
  <testcase classname="testCases.S001LogInCases" name="validEmailAndPassword" time="6.310"/>
  <system-out/>
</testsuite> <!-- testCases.S001LogInCases -->
